lib/driverProviders/driverProvider.ts

```diff
@@ -68,22 +67,22 @@ export abstract class DriverProvider {
    * @public
    * @param webdriver instance
    */
-  quitDriver(driver: WebDriver): wdpromise.Promise<void> {
+  async quitDriver(driver: WebDriver): Promise<void> {
     let driverIndex = this.drivers_.indexOf(driver);
     if (driverIndex >= 0) {
       this.drivers_.splice(driverIndex, 1);
     }
 
     if (driver.getSession() === undefined) {
-      return wdpromise.when(undefined);
+      return Promise.resolve(undefined);
     } else {
       return driver.getSession()
```

Comment:
This can now be `const session = await driver.getSession()`

Reply:
Fixed.